apps/studio/components/interfaces/Settings/Database/DatabaseSettings/DatabaseSettings.tsx

```diff
@@ -97,150 +97,145 @@ const DatabaseSettings = () => {
-                    copy
-                    disabled
-                    value={connectionInfo.db_name}
-                    label="Database name"
-                  />
-
-                  <Input
-                    className="input-mono"
-                    layout="horizontal"
-                    readOnly
-                    copy
-                    disabled
-                    value={connectionInfo.db_port.toString()}
-                    label="Port"
-                  />
-
-                  <Input
-                    layout="horizontal"
-                    className="input-mono table-input-cell text-base"
-                    readOnly
-                    copy
-                    disabled
-                    value={connectionInfo.db_user}
-                    label="User"
-                  />
-
-                  <Input
-                    className="input-mono"
-                    layout="horizontal"
-                    disabled
-                    readOnly
-                    value={
-                      state.selectedDatabaseId !== projectRef
-                        ? '[The password for your primary database]'
-                        : '[The password you provided when you created this project]'
-                    }
-                    label="Password"
-                  />
-                </>
-              )}
-            </Panel.Content>
-          </Panel>
-        </section>
-
-        <DatabaseConnectionString />
-
-        <ResetDbPassword disabled={isLoading || isError} />
-      </div>
+                ) : subscription?.plan.id === 'pro' && subscription?.usage_billing_enabled ? (
+                  <li>
+                    <Link href={`/org/${organization?.slug}/billing?panel=subscriptionPlan`}>
+                      <a className="text-foreground underline">Disable your Spend Cap</a>
+                    </Link>{' '}
+                    to allow your project to auto-scale and expand beyond the 8GB database size
+                    limit
+                  </li>
+                ) : null}
+              </ul>
+            </AlertDescription_Shadcn_>
+            <div className="mt-4 flex items-center space-x-2">
+              <Button type="default" onClick={() => setShowConfirmationModal(true)}>
+                Disable read-only mode
+              </Button>
+              <Button asChild type="default" icon={<IconExternalLink />}>
+                <Link
+                  href="https://supabase.com/docs/guides/platform/database-size#disabling-read-only-mode"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  Learn more
+                </Link>
+              </Button>
+            </div>
+          </Alert_Shadcn_>
+        )}
+
+        <Panel
+          title={
+            <div className="w-full flex items-center justify-between">
+              <h5 key="panel-title" className="mb-0">
+                Connect to your database directly
```

Comment:
I think we should combine the `Connect to your database directly` and `Connection string` panels. It should be clear to the user that there are 2 ways of connecting to the db - pooler and direct. That way we can add the `resolves to ipv4` and the deprecation notice to the direct connection panel. Right now if user just sees the `Connect to your database directly` panel, they may not see the deprecation notice below or think it may not apply to them.

Reply:
![image](https://github.com/supabase/supabase/assets/19742402/9f7c585b-5088-43d8-9d39-41267d68e025)